packages/sanity/src/core/config/prepareConfig.tsx

```diff
@@ -217,6 +220,11 @@ export function prepareConfig(
         types: schemaTypes,
       })
 
+      if (process.env.SANITY_STUDIO_SCHEMA_DESCRIPTOR) {
+        const sync = DESCRIPTOR_CONVERTER.get(schema)
```

Comment:
Should we add some timing here to track perf impact of doing this for large schemas?

Reply:
Something like that, yes. console.time/timeEnd would probably be very spammy. Not sure what a reasonable value of `X` would be 😅  (when we release this unflagged we can collect real-world numbers if we'd like)